src/cladetime/cladetime.py

```diff
@@ -78,13 +79,54 @@ def __init__(self, sequence_as_of=None, tree_as_of=None):
         else:
             self.url_ncov_metadata = None
 
+    @property
+    def sequence_as_of(self) -> datetime:
+        return self._sequence_as_of
+
+    @sequence_as_of.setter
+    def sequence_as_of(self, date) -> None:
+        """Set the sequence_as_of attribute."""
+        if date is None:
+            sequence_as_of = datetime.now()
+        sequence_as_of = self._validate_as_of_date(date)
+        utc_now = datetime.now().replace(tzinfo=timezone.utc)
```

Comment:
might this actually end up pointing to a past time?

```
>>> now = datetime.now()
>>> utc_now = now.replace(tzinfo=timezone.utc)
>>> # Create a timezone object for Eastern Time (UTC-4)
>>> eastern_tz = timezone(timedelta(hours=-4))
>>> eastern_now = now.replace(tzinfo=eastern_tz)
>>> 
>>> now
datetime.datetime(2024, 10, 10, 13, 50, 2, 440530)
>>> utc_now
datetime.datetime(2024, 10, 10, 13, 50, 2, 440530, tzinfo=datetime.timezone.utc)
>>> eastern_now
datetime.datetime(2024, 10, 10, 13, 50, 2, 440530, tzinfo=datetime.timezone(datetime.timedelta(days=-1, seconds=72000)))
>>> 
>>> eastern_now > utc_now
True
```

It seems like `replace` here does not calculate the time corresponding to "now" in UTC...

Reply:
Maybe the operation we want is
```
utc_now = datetime.now(tz=timezone.utc)
```